lib/activerecord/delay_touching.rb

```diff
@@ -76,8 +76,13 @@ def self.apply
 
     # Touch the specified records--non-empty set of instances of the same class.
     def self.touch_records(attr, klass, records)
-      attributes = records.first.send(:timestamp_attributes_for_update_in_model)
+      # Although we're now setting the default timestamp column upstream, we'll still want to grab the default attributes here.
```

Comment:
Would be curious to see this covered in a test or something, but 👍

Reply:
I'm honestly unsure what a decent test here would look like, but open to any suggestions. This change is effectively what made the failing spec since 5.1 work in the first place

Reply:
Yeah, sounds good. I wasn't sure which if the changes was specifically for that issue.